eng/testing/tests.mobile.targets

```diff
@@ -139,6 +139,8 @@
       <Optimized>true</Optimized>
       <MainLibraryFileName Condition="'$(MainLibraryFileName)' == ''">AppleTestRunner.dll</MainLibraryFileName>
       <_MobileIntermediateOutputPath Condition="'$(RunAOTCompilation)' == 'true'">$(IntermediateOutputPath)mobile</_MobileIntermediateOutputPath>
+      <!-- <ForceInvariant>$(InvariantGlobalization)</ForceInvariant> -->
+      <ForceInvariant>true</ForceInvariant>
```

Comment:
Why is this need if we already have `InvariantGlobalization` ?

Reply:
It's just a reminder that we use still force the use of invariant mode at runtime https://github.com/dotnet/runtime/blob/main/src/tasks/AppleAppBuilder/Templates/runtime.m#L212-L213

Reply:
Why do we need to do that? We have the ICU pieces working, right?

@steveisok

Reply:
ICU is working, but not yet merged.

The ICU PR takes the runtime.m setting away, but I don't mind having `ForceInvariant` should we need it.

Reply:
It would make sense to uncomment line 142 and use that.

Reply:
Done